README.md

```diff
@@ -1,5 +1,13 @@
 # FreeRTOS AWS Reference Integrations
 
+## Update (05/13/2021)
```

Comment:
This needs to be updated to the date after the tag is released

Reply:
changed the date and add link to the tag